README.md

```diff
@@ -30,7 +30,7 @@ in a for-profit context, please consider supporting my open source work on
 * Construct templates using a familiar, easy-to-use syntax
 * Template code is compiled into your crate for [optimal performance][benchmarks]
 * Benefit from the safety provided by Rust's type system
-* Optional built-in support for Actix, Gotham, Iron, Rocket and warp web frameworks
+* Optional built-in support for Actix, Gotham, Iron, Rocket, warp, and tide web frameworks
```

Comment:
Nit: I'd like to keep all the frameworks in alphabetic order (where they already are). This happens in a bunch of other places too, can you please review them?

Reply:
I made this change in 3712761ac45ef1771d9f17a9afcc3cf57c064919 and also alphabetically sorted the calls and functions in generator.rs